share/revng/test/configuration/revng-qa/for-decompilation.yml

```diff
@@ -27,3 +27,4 @@ sources:
     prefix: share/revng/test/tests/analysis/Decompilation/clang/
     members:
       - linked-lists.c
+      - dual-case-switch.c
```

Comment:
Please put this under `x86-64` (like e.g. `segments-and-sections.c`), not  `clang`, unless this is strictly necessary. And if for some reason you really really need `clang` I'd like you to explain why.

Reply:
No specific reason, just thought we wanted it done with `clang` since I usually do that when crafting tests. Put it under the `x86-64` folder in https://github.com/revng/revng-qa/commit/7d9d32d70292d147d1eec204ac75342c74c3fd68